docs/website/content/v0.6/en/guides/upgrading/talos.md

```diff
@@ -0,0 +1,244 @@
+---
+title: Upgrading alos
+---
+
+Keeping the infrastructure up to date is important for a stable and secure environment.
+Talos makes the upgrade process simple and efficient.
+There are 2 ways of upgrading Talos in this guide.
+
+### Method 1: upgrade trough talosctl
+
+We have a dedicated upgrade command in the `talosctl` CLI.
+There are some other settings worth checking before upgrading.
+
+Let's start by verifying the current version.
+
+``` bash
+$ talosctl version
+Client:
+        Tag:         v0.6.0-beta.0
+        SHA:         d17eb3d4
+        Built:
+        Go version:  go1.14.6
+        OS/Arch:     linux/amd64
+
+Server:
+        NODE:        192.168.1.82
+        Tag:         v0.6.0-alpha.6
+        SHA:         568e3985
+        Built:
+        Go version:  go1.14.6
+        OS/Arch:     linux/amd64
+```
+
+As you can see in the above example, we're running v0.6.0-alpha.6.
+We're going to upgrade to the v0.6.0-beta.0 today using the `talosctl upgrade` command.
+As you can see in the example above, we're going to upgrade the node: 192.168.1.82.
+
+Please make sure you're connected to the right node, otherwise do a: `talosctl config node <IP or DNS>`.
+
+> To get the right image, you can checkout the [release page on github](https://github.com/talos-systems/talos/releases/)
+> See [additional information](#getting-the-right-version) for more info on getting the correct image.
+
+``` bash
+$ talosctl upgrade -i docker.io/autonomy/installer:v0.6.0-beta.0 -p
```

Comment:
These links should point to the new gchr.io repo instead of docker.io

Reply:
probably 0.6 should still be on `docker.io`

starting with 0.7 we have new registry location `ghcr.io/talos-systems/installer:v0.7.0-xyz`